mk/px4_targets.mk

```diff
@@ -42,7 +42,9 @@ EXTRAFLAGS += -DUAVCAN=1
 
 # Add missing parts from libc and libstdc++
 EXTRAFLAGS += -DHAVE_STD_NULLPTR_T=0
-EXTRAFLAGS += -I$(SKETCHBOOK)/libraries/AP_Common/missing
+
+# Hardcoded libraries/AP_Common/missing/cmath defines in "make" to retain the current behavior
+EXTRAFLAGS += -DHAVE_MATH_ISFINITE='1'
```

Comment:
Actually in `environ.mlk` it is defined `HAVE_CMATH_ISFINITE` and here `HAVE_MATH_ISFINITE`.

Which will be used to not enter inside this condition:
```
#if defined(HAVE_CMATH_ISFINITE) && !defined(HAVE_MATH_ISFINITE)
using std::isfinite;
#endif
```